lib/model/transaction.dart

```diff
@@ -130,8 +130,8 @@ class Transaction extends BaseEntity {
       recurring: json[TransactionFields.recurring] == 1 ? true : false,
       recurrencyType: json[TransactionFields.recurrencyType] as String?,
       recurrencyPayDay: json[TransactionFields.recurrencyPayDay] as int?,
-      recurrencyFrom: json[TransactionFields.recurrencyFrom] != null ? DateTime.parse (TransactionFields.recurrencyFrom) : null,
-      recurrencyTo: json[TransactionFields.recurrencyTo] != null ? DateTime.parse (TransactionFields.recurrencyTo) : null,
+      recurrencyFrom: json[TransactionFields.recurrencyFrom] != null ? DateTime.parse(TransactionFields.recurrencyFrom) : null,
+      recurrencyTo: json[TransactionFields.recurrencyTo] != null ? DateTime.parse(TransactionFields.recurrencyTo) : null,
```

Comment:
This blank space was already been removed on #74, seems you're working on old files